From be8e6a30154d8450553c4770472e92a143a7ebaf Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jose.dapena@lge.com>
Date: Thu, 1 Mar 2018 17:50:38 +0100
Subject: [jumbo] Fix extensions/common jumbo build.

The helper namespace declarations for manifest_keys,
manifest_errors are declared both inside anonymous namespace
and extensions namespace. This is ambiguous and makes
compilation fail.

So move all the helper declarations out of the anonymous
namespace.

Change-Id: I778b81c8094b12842fd47ae982b6cf620d208671

diff --git a/extensions/common/manifest_handlers/permissions_parser.cc b/extensions/common/manifest_handlers/permissions_parser.cc
index 1bd9a1dc80df..d119aff3edec 100644
--- a/extensions/common/manifest_handlers/permissions_parser.cc
+++ b/extensions/common/manifest_handlers/permissions_parser.cc
@@ -29,11 +29,11 @@
 
 namespace extensions {
 
-namespace {
-
 namespace keys = manifest_keys;
 namespace errors = manifest_errors;
 
+namespace {
+
 struct ManifestPermissions : public Extension::ManifestData {
   ManifestPermissions(std::unique_ptr<const PermissionSet> permissions);
   ~ManifestPermissions() override;
diff --git a/extensions/common/manifest_handlers/sandboxed_page_info.cc b/extensions/common/manifest_handlers/sandboxed_page_info.cc
index 9574d31c8569..ea3c2f2d2918 100644
--- a/extensions/common/manifest_handlers/sandboxed_page_info.cc
+++ b/extensions/common/manifest_handlers/sandboxed_page_info.cc
@@ -19,11 +19,11 @@
 
 namespace extensions {
 
-namespace {
-
 namespace keys = extensions::manifest_keys;
 namespace errors = manifest_errors;
 
+namespace {
+
 const char kDefaultSandboxedPageContentSecurityPolicy[] =
     "sandbox allow-scripts allow-forms allow-popups allow-modals; "
     "script-src 'self' 'unsafe-inline' 'unsafe-eval'; child-src 'self';";
-- 
2.14.1

